project/AppDependencies.scala

```diff
@@ -2,14 +2,14 @@ import sbt._
 
 object AppDependencies {
 
-  val boostrapVersion = "7.19.0"
+  val boostrapVersion = "7.21.0"
```

Comment:
we may as well update play to 2.8.20 in plugins.sbt to get rid of the PR bot warning